### ISSUE TITLE
Review Console page

### ISSUE THREAD

**Issue by Garbee** [GoogleChrome/devtools-docs#222]
Look over the [Console](https://developer.chrome.com/devtools/docs/console) page.

Specifically:
* Update a couple outdated images.
* Make sure all examples are inline with the content.
* Clean up code examples so they aren't all single-line looking monstrosities.

**Comment by paulirish** [comment]
HTML!

**Comment by Garbee** [comment]
Viva le revolution!

**Comment by Garbee** [comment]
Ok, so while I am doing the conversion I decided to go ahead and update the content itself too. Things are way too wordy and repeated throughout the doc and my OCD just kicked in. So... This might take a bit longer than expected and will need a really good review when the PR is up.

**Comment by paulirish** [comment]
k.

**Comment by Meggin** [comment]
I can help with the review.

**Comment by Garbee** [comment]
PR #251 is in for this. We will pick up there. ENJOY!

:smiling_imp:
